setup.py

```diff
@@ -157,6 +157,9 @@ def run(self):
     },
     data_files=data_files,
     include_package_data=True,
+    package_data={
```

Comment:
I checked it by running the `make dist` target, but unfortunately the zip file will not contain these static files. The problem is that when `include_package_data` option is specified it will ignore the `package_data` option. To solve this problem try to create a `MANIFEST.in` beside the `setup.py` file with the following content:
```in
recursive-include build_dist/CodeChecker/lib/python3/plist_to_html/static *
```

Reply:
Thanks, adding `MANIFEST.in` as you suggest does indeed work as well. (I only tested with `python3 setup.py bdist`, for which both approaches work, curiously. But you are right, with my approach the source package was still lacking those files, while with the `MANIFEST.in` file both, source and binary packages are fixed.